hacks_mbof/settings.py

```diff
@@ -171,16 +171,24 @@
         'simple': {
             'format': '%(levelname)s %(name)s %(message)s'
         },
+        'ocellus_formatter': {
+            'format': '%(levelname)s %(asctime)s %(module)s %(message)s',
+        },
     },
     'handlers': {
         'console': {
             'class': 'logging.StreamHandler',
             'formatter': 'simple',
         },
+        'ocellus_handler': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'ocellus_formatter',
+        },
     },
     'loggers': {
         '': {
-            'handlers': ['console'],
+            # 'handlers': ['console'],
```

Comment:
Remove commented-out line.  Was `console` ever helpful?  Helpful enough to be immortalized in a comment?